networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -10,8 +10,10 @@ As a cluster administrator, you can configure network policy for additional netw
 
 [NOTE]
 ====
-You can specify multi-network policy for only macvlan additional networks.
-Other types of additional networks, such as ipvlan, are not supported.
+You can specify multi-network policy for SR-IOV and macvlan additional networks.
+Other types of additional networks, such as ipvlan, are not supported. 
+SR-IOV additional networks are technology preview and macvlan additional networks are fully supported. 
+SR-IOV additional networks are only supported with kernel network interface cards (NICs). They are not supported for DPDK applications. 
```

Comment:
Since there are support claims involved here, I think this makes sense as an `[IMPORTANT]` admonition. Also, I think breaking this out into an unordered list would make this easier to scan. Also, it reads more like a list than a paragraph with a thesis and supporting statements.

Reply:
See the comment above about _DPDK_.